config/examples/Creality/Ender-4/Configuration.h

```diff
@@ -491,15 +491,10 @@
 
   // If you are using a pre-configured hotend then you can use one of the value sets by uncommenting it
 
-  // Ender-4
-  #define DEFAULT_Kp 16.67
-  #define DEFAULT_Ki 1.12
-  #define DEFAULT_Kd 62.14
-
```

Comment:
Why remove the tuned Ender-4 defaults? You're also restoring Ultimaker defaults below which doesn't make sense here.